### PR TITLE
Feature/smar 91 

### PR DESCRIPTION
**Pull request by yulyaolshanska**
## Description changes:

- list of local doctors patients. Frontend and backend connection;
- search line. Frontend and backend connection;
- load more. Frontend and backend connection

## Issue ticket code (and/or) and link
[Link to JIRA ticket](https://smartdc23.atlassian.net/browse/SMAR-91)

### General
 
- [x] Assigned myself to the PR
- [x] Assigned the appropriate labels to the PR
- [x] Assigned the appropriate reviewers to the PR
- [ ] Updated the documentation
- [x] Performed a self-review of my code
- [x] Don't have "any" on my code
- [x] Don't have magic numbers
- [x] Compare only with constants not with strings
- [x] No ternary operator inside ternary operator
- [x] Don't have commented code
- [x] All links in on ENV files
- [x] Used camelCase for variables and functions
- [x] Functions is public only if it's used outside the class
- [x] No hardcoded values
- [x] Check your commit messages meet the [conventional commit format](https://www.conventionalcommits.org/en/v1.0.0/).

### Frontend
- [x] Components and business logic are separated
- [x] Colors, Font Size, and Font Name is on the theme or in the constants
- [x] No text in the components, use i18n approach
- [x] No inline styles
- [x] Imports are absolute
- [x] Attach a screenshot if PR has visual changes.

#### General view:

image

#### Search:
image

image

#### Load more button: 
image

### CODE REVIEW

File: src/components/general/styles.ts [hunk 1]
```diff
@@ -253,3 +254,31 @@ export const ButtonContainer = styled.div`
...
+`;
+
+export const LoadMoreButton = styled(SendButton)`
+  color: ${BLUE};
+  background: ${WHITE};
+  width: 131px;
+  height: 44px;
+  border: 1.5px solid ${BLUE};
+  border-radius: 4px;
+  font-size: 14px;
```
Comment (lyudad) [inline src/components/general/styles.ts]: constants

File: src/utils/functions/getGoogleOauthURL.ts [hunk 2]
```diff
@@ -1,7 +1,7 @@
 function getGoogleOauthURL(): string {
   const rootUrl = 'https://accounts.google.com/o/oauth2/v2/auth';
   const options = {
-    redirect_uri: 'http://localhost:5000/auth/google/redirect',
+    redirect_uri: 'http://localhost:5001/auth/google/redirect',
```
Comment (lyudad) [inline src/utils/functions/getGoogleOauthURL.ts:4]: why it here?

Comment (yulyaolshanska) [reply src/utils/functions/getGoogleOauthURL.ts:4]: Port 5000 already in use — MacOS Monterey issue. I can’t use this port

File: src/components/PatientItem/styles.ts [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+  display: flex;
+  align-items: center;
+  margin-bottom: 4px;
+`;
+
+export const ContactInfo = styled.p`
+  margin-left: 4px;
+  font-size: ${SMALL_FONT_SIZE};
+  line-height: 1.5;
+  color: #060d33;
```
Comment (lyudad) [inline src/components/PatientItem/styles.ts]: ?

File: src/components/PatientItem/styles.ts [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+  font-size: ${VERY_SMALL_FONT_SIZE};
+  line-height: 1.43;
+  color: #081972;
+`;
+
+export const LastAppointment = styled.p`
+  margin-bottom: 6px;
+  font-size: ${VERY_SMALL_FONT_SIZE};
+  line-height: 1.4;
+  color: #060d33;
```
Comment (lyudad) [inline src/components/PatientItem/styles.ts]: have you checked your code?

**Pull request closed without merging**